api/v1/azureappconfigurationprovider_types.go

```diff
@@ -64,13 +66,19 @@ type ConfigurationGenerationParameters struct {
 
 // AzureAppConfigurationKeyValueOptions defines the options of fetching key-values from AppConfiguration.
 type AzureAppConfigurationKeyValueOptions struct {
-	Selectors       []KeyValueSelector                     `json:"selectors,omitempty"`
+	Selectors       []KeyLabelSelector                     `json:"selectors,omitempty"`
 	TrimKeyPrefixes []string                               `json:"trimKeyPrefixes,omitempty"`
 	Refresh         *DynamicConfigurationRefreshParameters `json:"refresh,omitempty"`
 }
 
-// KeyValueSelector defines the filters when fetching the data from Azure AppConfiguration
-type KeyValueSelector struct {
+// AzureAppConfigurationFeatureFlagOptions defines the options of fetching feature flags from AppConfiguration.
+type AzureAppConfigurationFeatureFlagOptions struct {
+	Selectors []KeyLabelSelector `json:"selectors,omitempty"`
+	Refresh   *RefreshSettings   `json:"refresh,omitempty"`
+}
+
+// KeyLabelSelector defines the filters when fetching the data from Azure AppConfiguration
+type KeyLabelSelector struct {
```

Comment:
We will add `snapshotName` in the selector in the near future, so `KeyLabelSelector` is not adaptive enough. Do you mind rename this to `Selector`?